### DISCUSSION TITLE
Unable to Connect to WebSocket Server from Mobile Application (Using Flutter)

### DISCUSSION THREAD

**Discussion by Hasan188** [jchristn/WatsonWebsocket#136] [Q&A]
Hello.

I hope this message finds you well. I have encountered an issue while trying to connect the WebSocket server I created to my mobile application, which is developed using Flutter. Despite multiple attempts, I have been unsuccessful in establishing a connection between the Flutter packages and the server I implemented. I kindly request your assistance in resolving this matter. I am eager for your guidance.

The Flutter packages I am currently using as clients in my application are as follows:

[web_socket_channel](https://pub.dev/packages/web_socket_channel) (the most popular one)
[web_socket_client](https://pub.dev/packages/web_socket_client)
[obs_websocket](https://pub.dev/packages/obs_websocket)

I appreciate any help you can provide in understanding why the mobile application fails to connect to the server. Thank you very much in advance for your support.

Best regards,

**Comment by jchristn** [comment]
Hi @Hasan188 thanks for your note.  I unfortunately don't have any experience with Flutter or packages available for it.  I recommend testing with the Javascript code sample in the README:
![image](https://github.com/jchristn/WatsonWebsocket/assets/1421081/35d24351-5583-446a-a7b8-7186a9f4a48a)

Additionally you can test with the ```Test.Server``` and ```Test.Client``` projects to isolate and reproduce any issues.  Should you find an issue with the library, please open an issue on it and I'll take a look!  (Moving this to discussions)
